pkg/controllers/edgeconnect/secret/secret.go

```diff
@@ -80,3 +85,12 @@ func safeEdgeConnectCfg(cfg config.EdgeConnect) string {
 
 	return string(safe)
 }
+
+func createDefaultServiceAccount(token string) config.Secret {
```

Comment:
```suggestion
func createKubernetesApiSecret(token string) config.Secret {
```

Reply:
fixed.

Reply:
I don't see any change. Has something slipped the commit?

Reply:
my bad., fixed in a4f78fbd